res/txt/dsg/encounters/fields/elis/eisek_mob_hideout.xml

```diff
@@ -0,0 +1,213 @@
+	</p>
+	]]>
+	</htmlContent>
+	
+	<!-- CONFRONT -->
+	
+	<htmlContent tag="CONFRONT"><![CDATA[
+	<p>
+		You descend the steps into the building's cellar, following the sound of voices. It is surprisingly well lit and spacious inside. Unmarked crates and barrels line the walls, providing most of the seating, but there are some chairs upon which you see some familiar faces. One of those familiar faces is the long-winded donkey-boy who takes notice of you the instant to you enter the room.
+	</p>
+	<p>
+		[test.speechMasculine(You,)] he says, his face twisted in disgust as he extends an accusatory digit, [test.speechMasculine(Come to settle the score? You don't know anything about that [eisek.race] and what his kind is really like, what they did for generations to us common folk before they fell from favour.)]
+	</p>
+	<p>
+		You tell the donkey-boy, who you assume to be a leader of sorts, that you want them to leave Eisek alone. Your request is met by derisive laughter.
+	</p>
+	<p>
+		[test.speechMasculine(And why would we do that? We will not stand by while our homes are in danger!)]
+	</p>
+	<p>
+		What will do you to convince the mob?
+	</p>
+	]]>
+	</htmlContent>
+	
+	<!-- INTIMIDATE -->
+	<!-- TODO: Merge variants -->
+	<!-- TODO: Add variants for ferals who don't have hands (use teeth/hooves/horns/etc) and slimes -->
+	
+	<htmlContent tag="INTIMIDATE"><![CDATA[
+	<p>
+		#IF(pc.isFeral())
+			You bare your teeth
+		#ELSEIF(pc.getRace() == RACE_SLIME)
+			You increase your bulk by ingesting air
+		#ELSE
+			You crack your knuckles
+		#ENDIF
+		 menacingly and explain that if they don't honour your request, the next time you see them anywhere near Eisek you'll give them a thrashing they'll never forget. And with their history with the Enforcers, somehow you don't think you're going to be the one getting arrested.
+	</p>
+	<p>
+		A tough looking [npc.race] steps forward at your threat. [npc.speech(Think you're hot shit, huh? You don't scare me!)] [npc.she] says as [npc.she] charges forward!
+	</p>
+	]]>
+	</htmlContent>
+	
+	<!-- INTIMIDATE_ARCANE -->
+	
+	<htmlContent tag= "INTIMIDATE_ARCANE"><![CDATA[
+	<p>
+		#IF(pc.hasSpell(SPELL_FIREBALL))
+			You cast a ball of fire that flies right between the donkey-boy's legs and scorches the floor beneath his chair.
+		#ELSEIF(pc.hasSpell(SPELL_SLAM))
+			#IF(game.isSillyMode())
+				With a cry of [pc.speech(C'mon and slam!)] you punch the ground beneath you, causing
+			#ELSE
+				You cause
+			#ENDIF
+			 a small earthquake in the cellar. Dust rains down from the ceiling.
+		#ELSEIF(pc.hasSpell(SPELL_TELEKENETIC_SHOWER))
+			You lift a few small pebbles and make them orbit around the donkey-boy's head.
+		#ELSEIF(pc.hasSpell(SPELL_POISON_VAPOURS))
+			You summon a few wisps of a poisonous vapour and swirl them around the room.
+		#ELSEIF(pc.hasSpell(SPELL_ICE_SHARD))
+			You cast a shard of ice right that flies right between the donkey-boy's legs and freezes patch of floor beneath his chair.
```

Comment:
Should be like this I think: You cast a shard of ice that flies right between the donkey-boy's legs and freezes a patch of floor beneath his chair.